### ISSUE TITLE
Could you publish this to PyPI?

### ISSUE THREAD

**Issue by rbarrois** [kaniini/ircreactor#1]
Hi,

I was looking for an IRC protocol parsing library, and found this package; could you publish some version on PyPI for easy installation?

Thanks!

**Comment by kaniini** [comment]
I can, but it's not really production ready yet.  It also requires more lower-level knowledge of IRCv3 events than a high-level library (I assume you know this though).

**Comment by DanielOaks** [comment]
We can push it to PyPi, but keep in mind that the APIs of this library aren't yet stable and can change as we develop it. We do not recommend using this for production systems at this point. We've added some comments to the readme to say this more explicitly.

If you do find any issues with it though, please let us know!

**Comment by rbarrois** [comment]
Thanks for the feedback, this explains why it isn't available on PyPI yet :)

I'm looking for a "IRC protocol parser", basically the `ircreactor/envelope.py` module of this projet; if it's intended as an internal API, I'll look for another option though.

**Comment by DanielOaks** [comment]
Nah, it's not an internal API – we're just still writing and updating it as we go with our main project, [mammon](https://github.com/mammon-ircd/mammon). Don't want you to rely on it and then be confused if it breaks your code a few times before we finish with it, because we're changing APIs :)
